content/en/sheets/guide/rum_analysis.md

```diff
@@ -0,0 +1,85 @@
+
+1. Prepare a CSV file containing additional metadata. For example, your file might include columns for user IDs, premium status, and team information.
+1. Navigate to [Reference Tables][1] and click **New Reference Table +**.
+1. Upload the CSV file and define the primary key for the table.
+
+For more information, see the [Reference Tables][2] documentation.
+
+## Enriching RUM data with lookups
+
+1. In your Sheets interface, click **Add Lookup** at the top of the page.
+1. Select the Reference Table you uploaded and specify the common column, such as User ID, to match records.
+1. The Lookup function adds the relevant metadata from the Reference Table to your RUM sessions table.
+
+## Using pivot tables for analysis
+
+Pivot tables help you summarize and organize large datasets to find patterns and trends.
+
+1. From an existing spreadsheet that already has a table of data, click **Add Pivot Table**.
+2. In the **Rows** section, select the dimensions you want to analyze, such as the User name.
+3. In the **Calculations** section, choose the dimensions for calculations, like sum, average, count, min, and max.
+
+## Example use case: Analyzing premium user errors
+
+You have a web application and want to understand how errors are impacting your application's premium users. Follow these steps to analyze RUM sessions, enrich your data with additional user information, and gain insights into how your premium users are impacted by errors. Use sheets to help you make data-driven decisions to improve user experience and address issues more effectively.
+
+
+### Filter RUM Sessions
+1. Start from the [RUM Sessions Explorer][3] and build a query to filter sessions with two or more errors.
+
+     ```
+     @session.error.count:>=2 @session.type:user
+     ```
+     {{< img src="/sheets/guide/rum_analysis/rum_sessions_explorer.png" alt="RUM Session Explorer showing a query filtering @session.error.count:>=2 and @session.type:user" style="width:80%;" >}}
+1. Click **Open in Sheets** to create a table with the filtered sessions.
+
+### Enrich Data
+1. Upload a CSV file to a Reference Table with additional user metadata (such as premium status or team).
+1. Use the Lookup function in Sheets to add the premium status to your RUM sessions table.
+   {{< img src="/sheets/guide/rum_analysis/lookup_function.png" alt="Configure a Lookup with the external dataset and the additional metadata you want to pull from that dataset"style="width:80%;" >}}
+
+### Analyze with Pivot Tables
+1. Create a pivot table to count the number of errors per premium user.
+1. Summarize the data to understand the distribution of errors among premium and non-premium users.
+   {{< img src="/sheets/guide/rum_analysis/add_lookup_pivot_table.mp4" alt="Walkthrough of how to add a Lookup and create a pivot table in Sheets" video=true >}}
```

Comment:
Same comment about breaking this up into multiple videos or screenshots. (Not every step needs a screenshot if the step is pretty easy to figure out!)

Reply:
I'll add a separate tasker for this!